bin/run_FiberSpectrograph

```diff
@@ -21,8 +21,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 """The runner for the Fiber Spectrograph CSC."""
-import asyncio
+from lsst.ts.FiberSpectrograph.cli import execute_csc
```

Comment:
Is this file kept around for testing purposes (since the installed version will come from the entry point)?

Reply:
That's correct.

Reply:
Yes, it is the only way I know of to run locally (without an installed package). If there is a way to use entry points for this, I would love to know about it.

Reply:
`pip install -e .` does work (assuming the dependencies are available) to do a developer install that points to the working directory and I do use that for non-scons projects. The only downside is that to get it to work python installs an egg into the conda environment that then points back to your build tree. This is fine for local development but remembering to cleanup the egg (maybe `pip uninstall` will do it) is important if you don't want to be confused in the future.

Reply:
Yes, that's what EUPS could easily set up for you. It still requires the stub file in the `bin` dir though.